hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/server/SCMBlockProtocolServer.java

```diff
@@ -176,6 +176,10 @@ public void join() throws InterruptedException {
     auditMap.put("owner", owner);
     List<AllocatedBlock> blocks = new ArrayList<>(num);
     boolean auditSuccess = true;
+    LOG.info("Allocating blocks {} of size {}, with excludeList: " +
+        "datanodes = {}, pipelines = {}, containers = {}",
+        excludeList.getDatanodes(), excludeList.getPipelineIds(),
+        excludeList.getContainerIds());
```

Comment:
There are 5 placeholders, but only 3 values.  Number and size of blocks seems to be missing.

Also I think `Allocating {} blocks` would be more natural than `Allocating blocks {}`.

Reply:
Thanks @adoroszlai for catching this.